src/utils/logger/pino/pino.config.ts

```diff
@@ -0,0 +1,32 @@
+import type { LoggerOptions } from 'pino';
+
+import getLogBody from './helpers/get-log-body';
+import { type CustomLevels } from './pino.types';
+
+const isDevelopment = process.env.NODE_ENV === 'development';
+
+const LOGGER_CONFIG: LoggerOptions<CustomLevels> = {
+  level: isDevelopment ? 'trace' : 'info',
+  formatters: {
+    level(label, number) {
+      return { level: isDevelopment ? number : label };
+    },
+  },
+  // To add custom levels, update the corresponding type and add the level values here
+  // customLevels: {
+  //   silly: 5,
+  // },
+  browser: {
+    transmit: {
+      level: 'warn',
+      send: (level, logEvent) => {
+        navigator.sendBeacon(
+          '/api/log',
+          JSON.stringify(getLogBody(level, logEvent))
```

Comment:
Do we need to stringify the payload? it seems like [sendBeacon](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/sendBeacon) accepts data objects.

Reply:
It does not, unfortunately. We could send it as a blob but JSON is easier to work with.